packages/cli/src/commands/oracle/rates.ts

```diff
@@ -25,12 +25,13 @@ export default class GetRates extends BaseCommand {
     const res = this.parse(GetRates)
     const sortedOracles = await this.kit.contracts.getSortedOracles()
 
-    const rates = await sortedOracles.getRates(res.args.token)
+    const reports = await sortedOracles.getReports(res.args.token)
```

Comment:
Should the command also be renamed? `oracle:rates` -> `oracle:reports`

Reply:
Yes, good call